virtualenv.py

```diff
@@ -75,7 +76,7 @@
 
 # Return a mapping of version -> Python executable
 # Only provided for Windows, where the information in the registry is used
-if not is_win:
+if not is_win or not is_cpython:
```

Comment:
For example, this could be re-arranged to simply be `if is_win:` <following block of code> and `else:` the simple two lines returning an empty dict.

I am guessing you're wanting to add a `is_cpython` to differentiate ironpython in the future, as a python on Windows, from cpython on Windows.

However, virtualenv doesn't know ironpython yet exists? So there is no reason/*need* to really add a `is_cpython` flag in this particular PR. I think the extra logic could go in the one supporting IronPython itself.